example/app/example-app/routes/routeregistry.go

```diff
@@ -0,0 +1,39 @@
+package routes
```

Comment:
`routeregistry.go` instead of `routerregistry.go`?

Reply:
should be route registry